traffic_ops/traffic_ops_golang/steeringtargets/steeringtargets.go

```diff
@@ -159,7 +160,30 @@ func read(tx *sqlx.Tx, parameters map[string]string, user *auth.CurrentUser) ([]
 	return filteredTargets, nil, nil, http.StatusOK
 }
 
+func checkTypeValidity(typeID int, tx *sqlx.Tx) (bool, error) {
```

Comment:
I was thinking we might've had a function that did this exact same thing already, and while reviewing another PR I found it (in lib/go-tc): 
```
tc.ValidateTypeID(tx, s.TypeID, "server")
```
We should use that function instead.